content/learn/book/your-first-game/_index.md

```diff
@@ -0,0 +1,685 @@
+
+Here is a version of the game compiled to WebGL that you can play right away:  
+[https://mrk.sed.pl/bevy-showcase/breakout.html](https://mrk.sed.pl/bevy-showcase/breakout.html)
+
+You can take a look at the complete [source code of the game](https://github.com/bevyengine/bevy/blob/latest/examples/game/breakout.rs), but I suggest to follow along this tutorial first, since it will slowly build up to the example game one step at a time.
+
+So what are the elements of the game ?
+
+ * A paddle
+ * A ball
+ * Bricks
+ * Walls
+ * A score counter
+
+Let's start with the paddle:
+
+```rs
+use bevy::{
+    prelude::*,
+    render::pass::ClearColor,
+};
+
+fn main() {
+    App::build()
+        .add_plugins(DefaultPlugins)
+        .insert_resource(ClearColor(Color::rgb(0.9, 0.9, 0.9)))
+        .add_startup_system(setup.system())
+        .add_system(paddle_movement_system.system())
+        .run();
+}
+
+struct Paddle {
+    speed: f32,
+}
+
+enum Collider {
+    Solid,
+    Scorable,
+    Paddle,
+}
+
+fn setup(
+    commands: &mut Commands,
+    mut materials: ResMut<Assets<ColorMaterial>>,
+    asset_server: Res<AssetServer>,
+) {
+    commands
+        // cameras
+        .spawn(OrthographicCameraBundle::new_2d())
+        .spawn(UiCameraBundle::default())
+        // paddle
+        .spawn(SpriteBundle {
+            material: materials.add(Color::rgb(0.5, 0.5, 1.0).into()),
+            transform: Transform::from_xyz(0.0, -215.0, 0.0),
+            sprite: Sprite::new(Vec2::new(120.0, 30.0)),
+            ..Default::default()
+        })
+        .with(Paddle { speed: 500.0 })
+        .with(Collider::Paddle);
+}
+
+fn paddle_movement_system(
+    time: Res<Time>,
+    keyboard_input: Res<Input<KeyCode>>,
+    mut query: Query<(&Paddle, &mut Transform)>,
+) {
+    for (paddle, mut transform) in query.iter_mut() {
+        let mut direction = 0.0;
+        if keyboard_input.pressed(KeyCode::Left) {
+            direction -= 1.0;
+        }
+
+        if keyboard_input.pressed(KeyCode::Right) {
+            direction += 1.0;
+        }
+
+        let translation = &mut transform.translation;
+        // move the paddle horizontally
+        translation.x += time.delta_seconds() * direction * paddle.speed;
+        // bound the paddle
+        translation.x = translation.x.min(380.0).max(-380.0);
+    }
+}
+
+```
+
+The code might be a lot to take in at first glance, but don't worry. We will take it one section at a time.
+
+Before we do that however, let's see it in action!
+
+Create a new folder and project on your hard drive using this command:
+
+```bash
+cargo new my_breakout_game
+cd my_breakout_game
+```
+Edit Cargo.toml to add a dependency on the Bevy git repo as outline in the [getting started guide](/learn/book/getting-started/).
```

Comment:
```suggestion
Edit `Cargo.toml` (found in the `my_breakout_game` folder you just created) to add a dependency on the latest version of Bevy as outlined in the [getting started guide](/learn/book/getting-started/).
```

We should try not to point new users towards the frequently-broken `main` version of Bevy.